### PR TITLE
MI300X system tuning guide (BIOS/OS settings, System management sections)

### PR DESCRIPTION
**Pull request by peterjunpark**

### CODE REVIEW

File: docs/how-to/tuning-guides/mi300x/system-settings.rst [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+   The ``/proc/sys/kernel/numa_balancing`` file controls NUMA balancing in the
+   Linux kernel. If the value in this file is set to ``0``, the NUMA balancing
+   is disabled. If the value is set to ``1``, NUMA balancing is enabled.
+
+.. note::
+
+   Disabling NUMA balancing should be done cautiously and for
+   specific reasons, such as performance optimization or addressing
+   particular issues. Always test the impact of disabling NUMA balancing in
+   a controlled environment before applying changes to a production system.
```
Comment (peterjunpark) [inline docs/how-to/tuning-guides/mi300x/system-settings.rst:433]: Incorporated parts of the other section here.

File: docs/how-to/tuning-guides/mi300x.rst
Comment (peterjunpark) [inline docs/how-to/tuning-guides/mi300x.rst:1]: landing page wip

File: docs/how-to/tuning-guides/mi300x/system-settings.rst [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+
+.. _mi300x-bios-settings:
+
+System BIOS settings
+====================
+
+AMD EPYC 9004-based systems
+---------------------------
+For maximum MI300X GPU performance on systems with AMD EPYC™ 9004-series
+processors (codename “Genoa”) and AMI System BIOS, the following configuration
```
Comment (peterjunpark) [inline docs/how-to/tuning-guides/mi300x/system-settings.rst:35]:
```suggestion
processors (codename “Genoa”) and AMI System BIOS, the following configuration
```
Can we remove the codename?

File: docs/how-to/tuning-guides/mi300x/system-settings.rst [hunk 3]
```diff
@@ -0,0 +1,747 @@
...
+
+Deterministic clock
+'''''''''''''''''''
+
+Use the command ``rocm-smi --setperfdeterminism 1900`` to set the max clock
+speed up to 1900 MHz instead of the default 2100 MHz. This can reduce
+the chance of a PCC event lowering the attainable GPU clocks. This
+setting will not be required for new IFWI releases with the production
+PRC feature. Restore this setting to its default value with the
+``rocm-smi -r`` command.
```
Comment (peterjunpark) [inline docs/how-to/tuning-guides/mi300x/system-settings.rst:573]: Originally in the other section. Unsure where to put this.

File: docs/how-to/tuning-guides/mi300x/system-settings.rst [hunk 4]
```diff
@@ -0,0 +1,765 @@
...
+   :align: center
+   :alt: rocm-bandwidth-test unidirectional bandwidth
+
+Bidirectional bandwidth
+
+.. image:: ../../../data/how-to/tuning-guides/rbt-bidirectional-bandwidth.png
+   :align: center
+   :alt: rocm-bandwidth-test bidirectional bandwidth
+
+Acronyms
```
Comment (lpaoletti) [inline docs/how-to/tuning-guides/mi300x/system-settings.rst:662]: Don't think these are acronyms - better title might be Glossary

**Comment by peterjunpark** [comment]
Need redirects from `how-to/tuning-guides/*` to `how-to/system-optimization/*`

**Pull request merged**